src/structures/CommandHandler.ts

```diff
@@ -358,9 +359,18 @@ export class CommandHandler
 				});
 
 				this.client.webhook.error('CommandError', error);
-				message.reply(
-					'**an error occured, but rest assured! It has already been reported and will be fixed in no time!**',
-				).catch(() => null);
+				if (error instanceof WeebError)
+				{
+					message.reply(
+						`weeb.sh reported an error, try again (later)\n\`${error.message}\``,
```

Comment:
A more appropriate message would be:

> `weeb.sh` reported an error. The service might be unavailable currently.

Perhaps change `unavailable` with `unreacheable`.